docs/resources/new_in_doc.md

```diff
@@ -6,6 +6,36 @@ description: Overview of major recent additions to Ibexa DXP documentation.
 
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## November 2024
+
+### Infrastructure and maintenance
+
+- Added [v4.6.13 to v4.6.14 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.6/update_from_4.6/#v4614) which include security fixes.
+- Added [v3.3.40 to v3.3.41 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_3.3/update_from_3.3/#v3341) which include security fixes.
+
+### Content management
+
+- Added [AI Actions documentation](https://doc.ibexa.co/en/master/ai_actions/ai_actions/)
+
+### Search
+
+- Added [new `IsBookmarked` location criterion](https://doc.ibexa.co/en/master/search/criteria_reference/isbookmarked_criterion/)
+- [`IsUserEnabled` is now available on Solr and Elastisearch](https://doc.ibexa.co/en/master/search/criteria_reference/isuserenabled_criterion/)
+
+### Documentation
+
+- When searching from the top bar, you may see, at the bottom of the drop-down suggestion list, a link to a page to see more results.
```

Comment:
```suggestion
- When you search using the top bar, you can see a link to a page with further results at the bottom of the drop-down suggestion list.
```

Reply:
@julitafalcondusza The link is not always there. Maybe I should add in which condition it appears:
```suggestion
- When you search using the top bar, if there is more than the 10 listed results, you can see a link to a page with further results at the bottom of the drop-down suggestion list.
```

Reply:
"If there **are** ..."